### PR TITLE
[RFC] Adding VERBOSITY_LEVEL env var; Add to helm template

### PR DESCRIPTION
**Pull request by draychev**
I realize that we have great logging (and improving), but no easy way to enable that on the actual AGIC pod.

This PR adds:
 - `APPGW_VERBOSITY_LEVEL` environment variable to the Docker container
 - `verbosityLevel` to the Helm templates
 - read both - `APPGW_VERBOSITY_LEVEL` over `verbosityLevel` CLI flag

When AGIC starts - we see:
- `Using verbosity level 5 from environment variable APPGW_VERBOSITY_LEVEL`
or 
- `Using verbosity level 1 from CLI flag verbosity`

Having this environment variable would enable customers to easily bump verbosity to `5` so that they can see (sanitized) App Gateway config / JSON being applied to ARM. (w/ `kubectl logs ...`)

### CODE REVIEW

File: pkg/environment/environment.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ const (
 	WatchNamespaceVarName = "KUBERNETES_WATCHNAMESPACE"
 	// UsePrivateIPVarName is the name of the USE_PRIVATE_IP
 	UsePrivateIPVarName = "USE_PRIVATE_IP"
+	// VerbosityLevel sets the level of glog verbosity should the CLI argument be blank
+	VerbosityLevel = "APPGW_VERBOSITY_LEVEL"
```
Comment (akshaysngupta) [inline pkg/environment/environment.go]: `VerbosityLevelVarName`

File: pkg/environment/environment.go [hunk 2]
```diff
@@ -58,7 +62,7 @@ func ValidateEnv(env EnvVariables) {
 	}
 
 	if env.WatchNamespace == "" {
-		glog.Info("KUBERNETES_WATCHNAMESPACE is not set. Watching all available namespaces.")
+		glog.V(1).Infof("%s is not set. Watching all available namespaces.", VerbosityLevel)
```
Comment (akshaysngupta) [inline pkg/environment/environment.go]: s/VerbosityLevel/WatchNamespaceVarName

Comment (draychev) [reply pkg/environment/environment.go]: Thanks for catching that!

**Comment by draychev** [comment]
@akshaysngupta  Thank you!
PTAL

**Pull request merged**